ee-9/feature-pack/pom.xml

```diff
@@ -91,6 +91,7 @@
         <version.org.jboss.weld.weld>4.0.2.Final</version.org.jboss.weld.weld>
         <version.org.jboss.weld.weld-api>4.0.SP1</version.org.jboss.weld.weld-api>
         <version.org.jberet>2.0.0.Final</version.org.jberet>
+        <version.org.wildfly.transaction.client>2.0.0.Final</version.org.wildfly.transaction.client>
         <version.org.wildfly.security.jakarta.elytron-ee>2.0.0.Beta1</version.org.wildfly.security.jakarta.elytron-ee>
```

Comment:
I think the version property should contain "jakarta" to distinguish between the version for the existing counterpart.

Reply:
Just in case your comment relates to us adding "jakarta" to the Elytron override that was more because we changed the groupId for the project we split out:

https://github.com/wildfly-security/wildfly-elytron-ee/blob/2.x/pom.xml#L32

Reply:
okay, let's stick with the original property name.